index.html

```diff
@@ -161,6 +161,52 @@ <h2>A project for first-time contributions</h2>
 
                 <!-- ________ William's card END ________  -->
 
+		<!-- ________ Abbas's card START ________  -->
```

Comment:
Thank you for your PR . It looks great!

However, it seems like you have pasted your card (code) in some random location in the file. As outlined in the documentation, your code should be placed at the bottom of this _index.html_ file just above the line
```
 <!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
```

The reason for this is that it makes all subsequent merges cleaner without any possible merge conflicts. Moreover, it gives a clear way to understand the changes being made to this file as it grows in size.

Could you please move your code above the line I just mentioned and as outlined in the documentation?